applications/openshift/master/file_permissions_cni_conf_not_s390x/tests/ocp4/e2e.yml

```diff
@@ -0,0 +1,3 @@
+---
+# This will fail until OpenShift 4.14 is released and used by CI.
```

Comment:
Specifically - this will fail until https://issues.redhat.com//browse/OCPBUGS-22995 is landed in a 4.14 branch and released as a z-stream (which explains why this is still failing even though we're using 4.14 in CI).